bears/haml/HamlLintBear.py

```diff
@@ -0,0 +1,367 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.GemRequirement import GemRequirement
+import yaml
+
+
+@linter(executable='haml-lint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+) \[(?P<severity>W|E)\] '
+                     r'(?P<message>.*)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
+                      'W': RESULT_SEVERITY.NORMAL})
+class HamlLintBear:
+    """
+    Checks code using ``haml-lint`` for
+    HAML-specific style and lint checks
+    to ensure clean and readable HAML code.
+    """
```

Comment:
Those double `checks` in the doc string feel a bit strange. You could also allow those lines to get a bit longer ;)